tests/pytests/test_mp2.py

```diff
@@ -1,327 +1,424 @@
-        'MP2 TOTAL ENERGY':                     -76.22584631996111,
-        'SCS-MP2 CORRELATION ENERGY':            -0.20521117465845,
-        'SCS-MP2 TOTAL ENERGY':                 -76.22200064599551,
+    "conv": {
+        "HF TOTAL ENERGY": -76.01678947133706,
+        "MP2 SAME-SPIN CORRELATION ENERGY": -0.05268120425816,
+        "MP2 OPPOSITE-SPIN CORRELATION ENERGY": -0.15637564436589,
+        "MP2 CORRELATION ENERGY": -0.20905684862405,
+        "MP2 TOTAL ENERGY": -76.22584631996111,
+        "SCS-MP2 CORRELATION ENERGY": -0.20521117465845,
+        "SCS-MP2 TOTAL ENERGY": -76.22200064599551,
     },
 }  # yapf: disable
-for mp2type in ['df', 'conv']:
-    _ref_h2o_ccpvdz[mp2type]['SCF TOTAL ENERGY'] = _ref_h2o_ccpvdz[mp2type]['HF TOTAL ENERGY']
-    _ref_h2o_ccpvdz[mp2type]['CURRENT REFERENCE ENERGY'] = _ref_h2o_ccpvdz[mp2type]['HF TOTAL ENERGY']
-    _ref_h2o_ccpvdz[mp2type]['5050SCS-MP2 CORRELATION ENERGY'] = (
-        0.5 * (_ref_h2o_ccpvdz[mp2type]['MP2 SAME-SPIN CORRELATION ENERGY'] +
-               _ref_h2o_ccpvdz[mp2type]['MP2 OPPOSITE-SPIN CORRELATION ENERGY']))
-    _ref_h2o_ccpvdz[mp2type]['5050SCS-MP2 TOTAL ENERGY'] = _ref_h2o_ccpvdz[mp2type][
-        '5050SCS-MP2 CORRELATION ENERGY'] + _ref_h2o_ccpvdz[mp2type]['HF TOTAL ENERGY']
-
-
-@pytest.mark.parametrize("inp", [
-    pytest.param({'name': 'Mp2', 'custom': 'SCS-MP2', 'options': {'mp2_type': 'df'}}, id='mp2 (df)'),
-    pytest.param({'name': 'Mp2', 'custom': 'MP2', 'options': {'mp2_type': 'conv'}}, id='mp2 (conv)'),
-    pytest.param({'name': 'Mp2', 'custom': 'SCS-MP2', 'options': {'mp2_type': 'df', 'mp2_os_scale': 1.2, 'mp2_ss_scale': 0.33333333}}, id='explicit scs mp2 (df)'),
-    pytest.param({'name': 'Mp2', 'custom': 'SCS-MP2', 'options': {'mp2_type': 'conv', 'os_scale': 1.2, 'ss_scale': 0.33333333}}, id='explicit scs mp2 (conv)'),
-    pytest.param({'name': 'Mp2', 'custom': '5050SCS-MP2', 'options': {'mp2_type': 'df', 'mp2_os_scale': 0.5, 'mp2_ss_scale': 0.5}}, id='user-def scs mp2 (df)'),
-    pytest.param({'name': 'Mp2', 'custom': '5050SCS-MP2', 'options': {'mp2_type': 'conv', 'os_scale': 0.5, 'ss_scale': 0.5}}, id='user-def scs mp2 (conv)'),
-])  # yapf: disable
+for mp2type in ["df", "conv"]:
+    _ref_h2o_ccpvdz[mp2type]["SCF TOTAL ENERGY"] = _ref_h2o_ccpvdz[mp2type]["HF TOTAL ENERGY"]
+    _ref_h2o_ccpvdz[mp2type]["CURRENT REFERENCE ENERGY"] = _ref_h2o_ccpvdz[mp2type]["HF TOTAL ENERGY"]
+    _ref_h2o_ccpvdz[mp2type]["5050SCS-MP2 CORRELATION ENERGY"] = 0.5 * (
+        _ref_h2o_ccpvdz[mp2type]["MP2 SAME-SPIN CORRELATION ENERGY"]
+        + _ref_h2o_ccpvdz[mp2type]["MP2 OPPOSITE-SPIN CORRELATION ENERGY"]
+    )
+    _ref_h2o_ccpvdz[mp2type]["5050SCS-MP2 TOTAL ENERGY"] = (
+        _ref_h2o_ccpvdz[mp2type]["5050SCS-MP2 CORRELATION ENERGY"] + _ref_h2o_ccpvdz[mp2type]["HF TOTAL ENERGY"]
+    )
+
+
+@pytest.mark.parametrize(
+    "inp",
+    [
+        pytest.param({"name": "Mp2", "custom": "SCS-MP2", "options": {"mp2_type": "df"}}, id="mp2 (df)"),
```

Comment:
Why "Mp2" here?

Reply:
I like to throw odd capitalization patterns in to test upper/lower logic. This `test_scsmp2` test hasn't actually changed yet -- it just got caught by the formatter.

The main `test_mp2_module` I want to add  another check on `SCS-MP2 TOTAL/CORL ENERGY` present and `CUSTOM SCS-MP2 TOTAL/CORL ENERGY` absent. Then another parameterization to call `custom-mp2` with ss/os_scale and check the CUSTOM vars. Then this `test_scsmp2` can die.